build/vsts_build.yaml

```diff
@@ -725,6 +765,14 @@ phases:
     clean: true
     submodules: true
     
+  - task: UseDotNet@2
+    displayName: 'Use .NET Core sdk'
+    inputs:
+      packageType: sdk
+      version: 3.0.x
+      includePreviewVersions: true
```

Comment:
Did we make sure that this does not create too much bloat on windows machines as it does on linux? 
If it doesn't we should enable the occasional cleaning tasks.

Basically I'd recommend ensuring that storage sense  https://support.microsoft.com/en-us/help/12425/windows-10-free-up-drive-space is enabled on all of the DDNuGetWindows machines.

Reply:
@zivkan Please make sure this is happening.